component/src/main/java/io/siddhi/extension/io/grpc/sink/GrpcCallSink.java

```diff
@@ -0,0 +1,256 @@
+                                "each RPC will not exceed this number even if service config may allow a higher " +
+                                "number." ,
+                        type = {DataType.INT},
+                        optional = true,
+                        defaultValue = "5"),
+                @Parameter(name = "max.hedged.attempts",
+                        description = "Sets max number of hedged attempts. The total number of hedged attempts for " +
+                                "each RPC will not exceed this number even if service config may allow a higher " +
+                                "number." ,
+                        type = {DataType.INT},
+                        optional = true,
+                        defaultValue = "5"),
+                @Parameter(name = "retry.buffer.size",
+                        description = "Sets the retry buffer size in bytes. If the buffer limit is exceeded, no " +
+                                "RPC could retry at the moment, and in hedging case all hedges but one of the same " +
+                                "RPC will cancel." ,
+                        type = {DataType.LONG},
+                        optional = true,
+                        defaultValue = "16777216"),
+                @Parameter(name = "per.rpc.buffer.size",
+                        description = "Sets the per RPC buffer limit in bytes used for retry. The RPC is not " +
+                                "retriable if its buffer limit is exceeded." ,
+                        type = {DataType.LONG},
+                        optional = true,
+                        defaultValue = "1048576"),
+                @Parameter(name = "channel.termination.waiting.time",
+                        description = "The time in seconds to wait for the channel to become terminated, giving up " +
+                                "if the timeout is reached." ,
+                        type = {DataType.LONG},
+                        optional = true,
+                        defaultValue = "5"),
+                @Parameter(name = "max.inbound.message.size",
+                        description = "Sets the maximum message size allowed to be received on the channel in bytes" ,
+                        type = {DataType.LONG},
+                        optional = true,
+                        defaultValue = "4194304"),
+                @Parameter(name = "max.inbound.metadata.size",
+                        description = "Sets the maximum size of metadata allowed to be received in bytes" ,
+                        type = {DataType.LONG},
+                        optional = true,
+                        defaultValue = "8192"),
+        },
+        examples = {
+                @Example(
+                        syntax = "@sink(type='grpc-call', " +
+                                "url = 'grpc://194.23.98.100:8080/EventService/process', " +
+                                "sink.id= '1', @map(type='json')) "
```

Comment:
sink.id not explained in the param list !!! 

Have the example with grpc-call and grpc-call-response